internal/app/service.go

```diff
@@ -167,3 +165,22 @@ func (ps *AppService) DeleteApp(organizationID, appID string) error {
 
 	return nil
 }
+
+func CheckAppId(appId string) error {
```

Comment:
moved here from ./app.go since it was all that was left when the models were moved.